src/main/java/com/iota/iri/storage/LocalSnapshotsPersistenceProvider.java

```diff
@@ -0,0 +1,81 @@
+package com.iota.iri.storage;
+
+import com.iota.iri.conf.IotaConfig;
+import com.iota.iri.model.LocalSnapshot;
+import com.iota.iri.model.persistables.SpentAddress;
+import com.iota.iri.storage.rocksDB.RocksDBPersistenceProvider;
+import com.iota.iri.utils.Pair;
+
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Abstraction for localsnapshots-db persistence provider.
+ */
+public class LocalSnapshotsPersistenceProvider {
+    private PersistenceProvider provider;
+    private IotaConfig config;
+
+    public LocalSnapshotsPersistenceProvider(IotaConfig config){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4082603)